pkg/collectors/infrastructure.go

```diff
@@ -0,0 +1,93 @@
+package collectors
+
+import (
+	"context"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/watch"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/kube-state-metrics/pkg/metric"
+	"k8s.io/kube-state-metrics/pkg/version"
+
+	"github.com/golang/glog"
+
+	configv1 "github.com/openshift/api/config/v1"
+	configclient "github.com/openshift/client-go/config/clientset/versioned"
+)
+
+var (
+	infrastructureMetricFamilies = []metric.FamilyGenerator{
+		{
+			Name: "cluster_infrastructure_provider",
```

Comment:
Am I missing something as I don't see a prefix for any of these, e.g. openshift_ cluster_infrastructure_provider.

Reply:
@lilic we don't want to change the names of the metrics, as they already exist: https://github.com/openshift/cluster-kube-apiserver-operator/blob/master/pkg/operator/configmetrics/configmetrics.go